lang/cs/Org.Apache.REEF.Common/Telemetry/IMetricsSink.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Collections.Generic;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Common.Telemetry
+{
+    /// <summary>
+    /// Interface for metrics sink.
+    /// It is used to output IMRU metrics.
+    /// </summary>
+    [DefaultImplementation(typeof(DefaultMetricsSink))]
+    public interface IMetricsSink : IDisposable
+    {
+        void Sink(ISet<KeyValuePair<string, string>> metrics);
```

Comment:
Not sure whether the word `Sink` is the right verb for this method. Maybe `ProcessMetrics`? Or `ReceiveMetrics`? This could also be an `IObservable`, right?

Also, please add XMLDoc to this public interface method.